### ISSUE TITLE
Lombok installer won't find target in MyEclipse 2018.2 installation

### ISSUE THREAD

**Issue by Andrew-Poloni** [rzwitserloot/lombok#1849]
While running the Lombok installer and trying to integrate it into a Genuitec MyEclipse (v2018.2) installation the installer can't seem to find the file(s) it wants to target/modify.  If someone has a set of manual instructions to get Lombok integrated so MyEclipse will compile project code which includes Lombok annotations I'd be greatly appreciative! :) :+1:

**Comment by Maaartinus** [comment]
@Andrew-Poloni No idea what exactly your eclipse needs, but can't you point the installer to the install directory manually? The only modified file is `eclipse.ini` and the only change is adding the line
```
-javaagent:/your-path-to-eclipse/lombok.jar
```
just after `-vmargs`. BTW the installer shows this information when you click on the first blue link.

**Comment by Andrew-Poloni** [comment]
I did in fact point the installer to the directory manually; I suspect the problem is that the installer is looking for a `eclipse.ini` file.  For a MyEclipse installation there is no such file; the file is `myeclipse.ini` which I'm guessing the installer doesn't recognize (and I did point it to the directory where that file lives).

I appreciate that you can manually modify the (my)eclipse.ini file to get Lombok working, but the Lombok installer Eclipse docs lump MyEclipse in with Eclipse so I expected the installer to work with either.  I might suggest that perhaps either the installer code should be modified to look for a `myeclipse.ini` file as well or a note to the difference in the documentation might be in order.

**Comment by Maaartinus** [comment]
@Andrew-Poloni So, `myeclipse.ini` can be handled just like `eclipse.ini`, right? You did it manually and everything works for you now, right?

Disclaimer: I'm not a project owner and I'm not going to fix it myself. I was just trying to help you and make the problem clearer for someone who'll take care.

**Comment by rspilker** [comment]
For #1830 we added support for finding installer in LOCALAPPDATA. Since this is also a Genuitec product, there is a good chance that the problem is solved in the next release.

We'll do an edge release soon, please let us know if that fixes this problem. If not, please tell us the exact path to both the executable and the ini file.

**Comment by Andrew-Poloni** [comment]
As it turns out, the easy fix (for me, anyhow) was to simply rename `myeclipse.ini` to `eclipse.ini` and then point the Lombok installer to that file.  The Lombok installer then found the file and modified it nicely.  Afterwards I simply renamed the file back to `myeclipse.ini` and restared MyEclipse.  I already had the Lombok JAR file in the build path of my project, so when MyEclipse restarted all my Lombok-related annotation compilation errors went away. :+1: That said, I'm assuming that a change of the installer code to look for files named `myeclipse.ini` along with `eclipse.ini` will fix this issue. :)

I like the fact that the Lombok installer puts the absolute path to the Lombok JAR into the `.ini` file; i.e., the location of the Lombok JAR used to launch the Lombok installer.  I would have probable missed that little detail had I done it manually.

BTW, I should have provided more context originally: I'm running on CentOS v7.5.

**Comment by rspilker** [comment]
That is strange, we should also accept `myeclipse.ini`.

**Comment by Andrew-Poloni** [comment]
I just double-checked and the (Lombok v1.16.10) installer can't find the `myeclipse.ini` file in the directory where I installed MyEclipse (Bling) on CentOS v7.5 (~/Genuitec/MyEclipse/v2018_8_0).  Should I be using a later version of the Lombok JAR perhaps?

**Comment by Andrew-Poloni** [comment]
I just tried the latest version (v1.18.2) of the Lombok JAR file on CentOS v7.5 (Java v1.8.1_181) and the Lombok installer found both the `myeclipse` and `myeclipse.ini` file (after I directed it to the appropriate directory). (See attached.) My project has been using 1.16.10 for whatever reason and hasn't upgraded.  Looks like this was an issue you guys fixed some time ago ... thanks for looking into it.

![project lombok v1 18 2 - installer_003](https://user-images.githubusercontent.com/21042310/45038581-7f454e00-b02f-11e8-8a15-d389c0669d4c.png)
